test/models/rental_item_image_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RentalItemImageTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Might be nice to put a small test here, instead of having nothing. Though could be a placeholder if we have nothing to test yet.

Reply:
For now i left this in as a placeholder... We can add a todo to make sure we dont forget

```suggestion
  # TODO Training of first test
  # end
```